modules/MIPS/clinical_measures.php

```diff
@@ -227,6 +227,17 @@ function GenXmlMIPS(sNested) {
 
 
 function validateForm() {
+var d = document.forms[0];
+
+  FromDate = d.form_begin_date.value;
+  ToDate = d.form_target_date.value;
+
+  if(FromDate.length > 0 && ToDate.length > 0) {
+    if(FromDate > ToDate) {
+      alert("<?php echo xls('End date must be later than Begin date!'); ?>");
```

Comment:
use iziModalToast lib here to report msgs to the user http://izitoast.marcelodolce.com/ and see some sample usage here https://github.com/LibreHealthIO/lh-ehr/pull/1395/commits/db8f6b670c555193d15de3f866ec479c6a89fb53

Reply:
@muarachmann Okay ! will replace the alert statement.